pkg/machines/machines.less

```diff
@@ -5,10 +5,6 @@
 @import (less) "../../node_modules/c3/c3.css";
 
 // For patternfly-react and @patternfly/react-console
-@import "~xterm/lib/xterm.css";
```

Comment:
Why you dropped this in this commit? Is it somehow related?

Reply:
We don't seem to need it, the component import what it needs.
But it should be separate commit indeed.